shortlister/model.py

```diff
@@ -93,15 +100,53 @@ def load_applicants(path: Path):
     files = path.glob("*.pdf")
     applicants = []
     for file in files:
-        name_parts = file.stem.split("_")
-        applicant = Applicant(
-            name=" ".join(name_parts[0:2]), cv=file, scores={}, notes=""
-        )
+        applicant = load_applicants_from_pdf(file)
         applicants.append(applicant)
+        
     sort_alpha(applicants)
     return applicants
 
 
+def load_applicants_from_pdf(file: Path):
+    """Create a single Applicant instance from candidate pack within the parsed in PDF file"""
+    try:
+        doc = pymupdf.open(file)
+    except ValueError:
+        print(f"Corrupt/incorrect pdf file: {file}")
```

Comment:
Note, if exception is raised here, the variable `doc` will not be set and line 118-141 below can't be executed.

Reply:
Now thinking about it, since now the default value for each applicant is ```<unretrievable>```, catching exceptions are not necessary anymore?